app/js/main.js

```diff
@@ -46,11 +46,15 @@ var width  = 640,
     btnLoop       = document.querySelector("#btn-loop"),
     playback      = document.querySelector("#playback"),
     btnPlayPause  = document.querySelector("#btn-play-pause"),
+    btnNext       = document.querySelector("#btn-next"),
```

Comment:
Change ID selectors to following format for clarity:

`#btn-frame-[location]` (e.g. `#btn-frame-next`)